presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -6296,21 +6331,21 @@ public void testInvalidMapUnionSum()
     {
         assertQueryFails(
                 "SELECT map_union_sum(x) from (select cast(MAP() as map<varchar, varchar>) x)",
-                ".*line 1:8: Unexpected parameters \\(map\\(varchar,varchar\\)\\) for function map_union_sum. Expected: map_union_sum\\(map\\(K,V\\)\\) K:comparable, V:nonDecimalNumeric.*");
+                ".*Unexpected parameters \\(map\\(varchar,varchar\\)\\) for function map_union_sum.*");
```

Comment:
Does C++ not print out the expected signatures ? Actually its a good practice to print them out..its very useful for the user for debugging. Do you see different error messages with side-car vs not ? We could create an issue for Prestissimo for the error message.

Reply:
Thanks for pointing it out, modified the error message and opened a fix in Velox here: https://github.com/facebookincubator/velox/pull/14049. Could you please take another look?